scripts/dps-spring-boot-gradle-plugin/README.md

```diff
@@ -0,0 +1,14 @@
+# Scripts for DPS Spring Boot Gradle Plugin
+
+These scripts make it easier to bulk upgrade a number of repos with the latest Spring Boot Gradle Plugin version
+
+Steps to run:
+
```

Comment:
one bit that is missing is that you need to have checked out the repo and have it named the same before running the script.

Reply:
Good point, addressed